how-to/common/public/apps.json

```diff
@@ -180,6 +180,32 @@
 		"images": [],
 		"tags": ["page", "openfin", "client", "interop", "fdc3", "contact"]
 	},
+	{
+		"appId": "fixed-panels",
+		"name": "fixed-panels",
+		"title": "Fixed panels",
```

Comment:
Should the title be Manager Portal (Panelled)
 or Manager Portal Extended

Reply:
I didnt want it to confuse sales by having 2 manager portal entries